README.md

```diff
@@ -30,6 +31,34 @@ Once the server is running, you can run the mock API tests for your SDK. See the
 
 You can check which scenarios are included in the package by referencing the package's [README](https://github.com/smartsheet/smartsheet-sdk-tests/blob/master/sdk_tests_package/README.md). This README includes information on how to run the server as well as descriptions of each scenario.
 
+## Running Tests Locally
+To run the tests locally you need to just set up a few scripts that start wiremock and then run the tests.
+
+### Example start wire mock shell script locally.
+```bash
+#!/usr/bin/env sh
+#This is just the install and start wiremock scripts combined. 
+set -e
+WIREMOCK_PACKAGE="./sdk_tests_package" #or path to sdk_tests_package directory
+cp -r "$WIREMOCK_PACKAGE" .
+WIREMOCK_PACKAGE_DIR='sdk_tests_package'
+
+# run wiremock
+(cd $WIREMOCK_PACKAGE_DIR; ./launch.sh &)
+
+# wait for wiremock to start
+sleep 10
+```
+
+### Running Tests Locally
+```bash
+#!/bin/bash
+
+cd ../smartsheet-csharp-sdk/mock-api-test-sdk-net60/
+dotnet build --configuration Release --no-restore
```

Comment:
on a fresh fork i had to run `dotnet restore` before this build would work

Reply:
I will add it to the script. Thanks for testing!

Reply:
OK @rzaracota should be better now. Feel free to take a look.

Reply:
Just manually verified these steps from a new clone!